papers/Sepideh_Alassi_Bertram_Winter/PyCells_for_an_Open_Semiconductor_Industry

```diff
@@ -0,0 +1,270 @@
+
+The aim of this project is developing interoperable layout generators with Python (PyCells), for H35, C35, and S35 technologies of ams AG, that can be used by any EDA-tool supporting OpenAccess database. The OpenAccess database will be further explained in the next section. For a certain set of parameters, PyCells should generate layouts identical to those generated by SKILL codes (PCells). Furthermore, extra interactive features such as stretching and auto-abutment are added to PyCells which are verified to be accurate and are optimized with respect to performance.        
+
+
+Development
+-----------
+
+OpenAceess [OpenAccess]_ has been established as standard for storing design data in the semiconductor industry and builds the foundation for interoperability of EDA-tools. OpenAccess is a file based database and manages logical design data (schematics) as well as artwork data (layout) for manufacturing. The reference implementation for accessing design data within the database is written in C++. The company Ciranova (now acquired by Synopsys) developed a Python wrapper [PythonAPI]_ for the OpenAccess C++ class library with the goal to access and modify design data through Python. Additionally, they provide an integrated development environment [SynopsysPyCellStudio]_ that enables interactive development and debugging of Python code, while the effects on the design data are shown directly in a separate window of the GUI.
+
+In our development, we have used Python API library to write Python codes which generate recurring layout structures based on a given set of parameters. Typical recurring structures are primitive devices that are used in integrated circuit design such as resistors, capacitors and transistors. The electrical properties of the mentioned devices are brought down to geometric dimensions (e. g. width and length) which serve as parameters for the Python scripts. The object oriented nature of Python matches very well with this task as shown in the following example:
```

Comment:
I do not understand what "brought down to" means in this context. Please clarify?